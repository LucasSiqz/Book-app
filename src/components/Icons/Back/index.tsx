const Back = () => (
  <svg
    width="16"
    height="16"
    viewBox="0 0 16 16"
    fill="none"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M15 8H1"
      stroke="black"
      strokeWidth="2"
      strokeLinecap="round"
      strokeLinejoin="round"
    />
    <path
      d="M8 15L1 8L8 1"
      stroke="black"
      strokeWidth="2"
      strokeLinecap="round"
      strokeLinejoin="round"
    />
  </svg>
)
export default Back
