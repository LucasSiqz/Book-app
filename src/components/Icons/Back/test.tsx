import { render } from 'utils/test-utils'

import Back from '.'

describe('<BackIcon />', () => {
  it('should render correctly', () => {
    const { container } = render(<Back />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
