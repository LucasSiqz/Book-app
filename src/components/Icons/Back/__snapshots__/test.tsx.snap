// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackIcon /> should render correctly 1`] = `
<svg
  fill="none"
  height="16"
  viewBox="0 0 16 16"
  width="16"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M15 8H1"
    stroke="black"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
  />
  <path
    d="M8 15L1 8L8 1"
    stroke="black"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
  />
</svg>
`
